### PR TITLE
[master] Zil 4827 evm reverts do not have correct error handling

### PR DESCRIPTION
**Pull request by moboware**
## Backward Compatibility

- [x] This is not a breaking change
- [ ] This is a breaking change

## Review Suggestion
Saeed check the test code
Steve check the lookupserver code changes

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: tests/EvmAcceptanceTests/test/ContractRevert.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const { expect } = require("chai");
```
Comment (its-saeed) [inline tests/EvmAcceptanceTests/test/ContractRevert.js:1]: seems not needed

File: tests/EvmAcceptanceTests/test/ContractRevert.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+describe("Revert Contract Deployment", function () {
+  let contract;
+  before(async function () {
+    contract = await web3_helper.deploy("Revert");
+  });
+
+
+  it("Will revert the contract", async function () {
```
Comment (its-saeed) [inline tests/EvmAcceptanceTests/test/ContractRevert.js:16]: can you please be more verbose? `Will revert the contract when something is done`

**Review by its-saeed — APPROVED** [review]
just a few comments

**Review by CSideSteve — APPROVED** [review]
LGTM

P.S. How do you know when Michel has edited a file ? because you can't find the code because of all the const keywords.

**Pull request merged**